addons/packages/vsphere-cpi/1.20.0/README.md

```diff
@@ -22,7 +22,7 @@ None
 | `vsphereCPI.password` | Required | Password used to access a vSphere endpoint. Default value is `null`. |
```

Comment:
Shouldn't the description be => default value is `""`, for all the required values